### DISCUSSION TITLE
Support 8K

### DISCUSSION THREAD

**Discussion by ryanwoodcox** [nomi-san/parsec-vdd#9] [Ideas]
Is there a way to support 8k resolution (even at a low 60hz)?

**Comment by nomi-san** [comment]
Follow [this guide](https://support.parsec.app/hc/en-us/articles/4423615425293-VDD-Advanced-Configuration) to add 8K resolution.

```
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Parsec\vdd\
- 0\
  - width:   7680
  - height:  4320
  - hz:      60
```
